### PR TITLE
python-flask-example-heroku materials

### PR DESCRIPTION
**Pull request by miguelgarcia**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-flask-example-heroku/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+venv
```
Comment (bzaczynski) [inline python-flask-example-heroku/.gitignore:1]: We can add the `__pycache__/` folder too.

File: python-flask-example-heroku/config.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import os
+
+
+class Config(object):
```
Comment (bzaczynski) [inline python-flask-example-heroku/config.py:4]: Use class declaration syntax form Python 3:

```suggestion
class Config:
```

File: python-flask-example-heroku/app.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import os
+from flask import Flask
+
+app = Flask(__name__)
+app.config.from_object(os.getenv("APP_SETTINGS", "config.DevelopmentConfig"))
+
+
+@app.route("/")
+def index():
+    return f"The configured secret key is {app.config.get('SECRET_KEY')}"
```
Comment (bzaczynski) [inline python-flask-example-heroku/app.py:10]: The latest version from the tutorial looks like this:

```suggestion
import os
from flask import Flask

app = Flask(__name__)
env_config = os.getenv("APP_SETTINGS", "config.DevelopmentConfig")
app.config.from_object(env_config)


@app.route("/")
def index():
    secret_key = app.config.get("SECRET_KEY")
    return f"The configured secret key is {secret_key}."
```

File: python-flask-example-heroku/config.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    DEBUG = False
+    TESTING = False
+    SECRET_KEY = os.getenv("SECRET_KEY", "this-is-read-from-env")
```
Comment (bzaczynski) [inline python-flask-example-heroku/config.py:7]: According to the tutorial, this should be the following:

```suggestion
    SECRET_KEY = os.getenv("SECRET_KEY", "this-is-the-default-key")
```

File: python-flask-example-heroku/config.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class StagingConfig(Config):
+    DEVELOPMENT = True
+    DEBUG = True
+
+
+class DevelopmentConfig(Config):
+    DEVELOPMENT = True
+    DEBUG = True
```
Comment (bzaczynski) [inline python-flask-example-heroku/config.py:21]: The rest is this:

```suggestion
class ProductionConfig(Config):
    pass


class StagingConfig(Config):
    DEBUG = True


class DevelopmentConfig(Config):
    DEBUG = True
    DEVELOPMENT = True
```

**Review by bzaczynski — CHANGES_REQUESTED** [review]
@miguelgarcia Please update the accompanying PR in the materials repository to reflect your changes in the tutorial. Thanks!

**Comment by bzaczynski** [comment]
lgtm

**Review by jaschmitt — APPROVED** [review]

**Pull request merged**